README.md

```diff
@@ -10,7 +10,7 @@
 
 Bring icons to your VS Code.
 
-<img src="https://raw.githubusercontent.com/robertohuertasm/vscode-icons/master/images/screenshot_folders.png" >
+![demo](/images/screenshot890x486.gif)
```

Comment:
Just to remember me the Github's max image size. I know that I can take it from the image itself but this way is faster in case I need to rework it (which I know it was going to be the case).